.NET/Microsoft.Recognizers.Definitions.Common/Portuguese/NumbersWithUnitDefinitions.cs

```diff
@@ -818,7 +819,7 @@ public static class NumbersWithUnitDefinitions
         };
       public static readonly Dictionary<string, string> AmbiguityFiltersDict = new Dictionary<string, string>
         {
-            { @"null", @"null" }
+            { @"\b\d+\s*\p{L}+$", @"((\d+(\s*\p{L}+[-—–-]|\p{L}+)\d+)|(((\p{L}|\d)[-—–-]\d+\s*|\p{L}\d+)\p{L}+))" }

```

Comment:
Can you explain this rule?

Reply:
it was added in English to filter expressions like "24703 L-202". I thought it could be useful in Portuguese too, but is not required by this PR, so it can be removed.

Reply:
No need to remove. I was just considering why as I couldn't understand it jsut reading the regexes. :)